src/db/migrate/20170422014344_add_quiz_id_to_question.rb

```diff
@@ -0,0 +1,5 @@
+class AddQuizIdToQuestion < ActiveRecord::Migration[5.0]
+  def change
+    add_column :questions, :quiz_id, :integer
+  end
+end
```

Comment:
Quizzes and questions are already linked in the db so why do you want to link them again?